<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>version_builder</class>
 <widget class="QMainWindow" name="version_builder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>487</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>版本生成器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>要打开的工程位置:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_project"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_openfile">
         <property name="text">
          <string>打开...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Resources/131.png</normaloff>Resources/131.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>要构建版本的位置:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_src_path">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse_loadp">
         <property name="text">
          <string>浏览...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Resources/130.png</normaloff>Resources/130.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>要接受的文件后辍:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_patterns"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_refresh_pattern">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 255);
font: 9pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Resources/135.png</normaloff>Resources/135.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_reset">
         <property name="text">
          <string>重置</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Resources/133.png</normaloff>Resources/133.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="tv_dir">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="indentation">
          <number>15</number>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <attribute name="headerVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>500</number>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>目录/文件</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>设置标记</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>压缩包文件</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_6" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>  附加压缩文件名:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_zip">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse_zipp">
         <property name="text">
          <string>浏览...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Resources/130.png</normaloff>Resources/130.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_sign">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_size">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>要保存版本的位置:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_save_path">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse_savep">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>浏览...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Resources/130.png</normaloff>Resources/130.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>版本号:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_version">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
         <property name="text">
          <string>1000</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_save">
         <property name="text">
          <string>构造版本</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Resources/134.png</normaloff>Resources/134.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="new_file"/>
   <addaction name="open_file"/>
   <addaction name="save_file"/>
   <addaction name="separator"/>
   <addaction name="build_version"/>
   <addaction name="about"/>
  </widget>
  <action name="new_file">
   <property name="icon">
    <iconset>
     <normaloff>Resources/132.png</normaloff>Resources/132.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="open_file">
   <property name="icon">
    <iconset>
     <normaloff>Resources/131.png</normaloff>Resources/131.png</iconset>
   </property>
   <property name="text">
    <string>打开工程</string>
   </property>
  </action>
  <action name="build_version">
   <property name="icon">
    <iconset>
     <normaloff>Resources/134.png</normaloff>Resources/134.png</iconset>
   </property>
   <property name="text">
    <string>构造版本</string>
   </property>
  </action>
  <action name="save_file">
   <property name="icon">
    <iconset>
     <normaloff>Resources/128.png</normaloff>Resources/128.png</iconset>
   </property>
   <property name="text">
    <string>保存工程</string>
   </property>
   <property name="toolTip">
    <string>保存工程</string>
   </property>
  </action>
  <action name="about">
   <property name="icon">
    <iconset>
     <normaloff>Resources/129.png</normaloff>Resources/129.png</iconset>
   </property>
   <property name="text">
    <string>关于软件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="version_builder.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_save</sender>
   <signal>released()</signal>
   <receiver>build_version</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>new_file</sender>
   <signal>triggered()</signal>
   <receiver>version_builder</receiver>
   <slot>on_new_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_file</sender>
   <signal>triggered()</signal>
   <receiver>version_builder</receiver>
   <slot>on_open_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>build_version</sender>
   <signal>triggered()</signal>
   <receiver>version_builder</receiver>
   <slot>save_result()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>about</sender>
   <signal>triggered()</signal>
   <receiver>version_builder</receiver>
   <slot>on_about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_openfile</sender>
   <signal>released()</signal>
   <receiver>open_file</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_file</sender>
   <signal>triggered()</signal>
   <receiver>version_builder</receiver>
   <slot>on_save_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_new_file()</slot>
  <slot>on_open_file()</slot>
  <slot>on_save_file()</slot>
  <slot>save_result()</slot>
  <slot>on_about()</slot>
 </slots>
</ui>
